File: code/code_example.py
"""排版示例：用数字 1、2、3、4 枚举不含重复数字的三位数。

改编自 CUMCM 模板 code/code_example.py；仅使用 Python 标准语法。
"""

for i in range(1, 5):
    for j in range(1, 5):
        for k in range(1, 5):
            if i != j and i != k and j != k:
                print(i, j, k)

File: code/code_example.m
% 排版示例：合并等高线与梯度方向图。
% 改编自 CUMCM 模板 code/code_example.m，补全原示例的输入变量。

[X, Y] = meshgrid(-2:0.2:2);
Z = X.^2 + Y.^2;
[U, V] = gradient(Z, 0.2, 0.2);

figure;
contour(X, Y, Z);
hold on;
quiver(X, Y, U, V);
hold off;
axis equal;
xlabel('x');
ylabel('y');
